azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/EventHubPartitionPump.java

```diff
@@ -211,7 +211,6 @@ public void onReceive(Iterable<EventData> events)
 		@Override
 		public void onError(Throwable error)
```

Comment:
>onError [](start = 14, length = 7)

ReceivePump behavior should also change. ReceivePump shuts down after delivering the error to the handler. Or maybe, PartitionManager can reset the handler on error.